src/Service/Payment/PaymentOptionService.php

```
@@ -0,0 +1,440 @@
+        if ($hasVisa && $hasMastercard) {
+            return $this->getIconPath('card');
+        }
+
+        // Otherwise, try to use the first available brand's icon
+        if (!empty($this->availableCardBrands)) {
+            return $this->getCardBrandIconPath($this->availableCardBrands[0]);
+        }
+
+        // Fallback to generic card icon
+        return $this->getIconPath('card');
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'getDynamicCardBrandsLogo'. (Unused Code Rules)

(UnusedPrivateMethod)